examples/scripts/thumbnails.mjs

```diff
@@ -63,5 +68,26 @@ async function takeScreenshots() {
         await browser.close();
     }
 }
-
-takeScreenshots();
+async function sleep(ms) {
+    return new Promise(resolve => {
+        setTimeout(resolve, ms);
+    });
+}
+async function main() {
+    console.log('Spawn server on', port);
+    // We need this kind of command:
+    // npx serve dist --config ../serve.json
+    // Reason: https://github.com/vercel/serve/issues/732
+    // (who *ever* thought that stripping .html was a good idea in the first place...)
+    const server = spawn('serve', ['dist', '-l', port, '--no-request-logging', '--config', '../serve.json']);
+    await sleep(1000); // give a second to spawn server
```

Comment:
is there a better way to do this?

Reply:
Ditching `serve` package for having strange/breaking peculiar default behaviour in favor of `http-server` would make the code cleaner, but I'm not sure who and for what reason picked `serve`, maybe it does something which we would miss with `http-server` again... so I just added a `serve` config to fix that behaviour so far - but I also don't like it. We could probably just use clean/normal JS API to spawn a server aswell: `import { createServer } from 'http'` (or express)